os/hosting-cloud-config-using-nginx.md

```diff
@@ -0,0 +1,21 @@
+## Hosting Cloud-Config Using Nginx
+
+One of the ways to host your `cloud-config` files is to use [nginx][nginx]. You can also use [http_sub_module][http_sub_module] which will allow you to use `$private_ipv4` and `$public_ipv4` substitution variables referenced in other documents. By default this module is enabled in official nginx packages and in most Linux distributions.
```

Comment:
The nginx HTTP server can be used serve `could-config` files to booting CoreOS machines. With the addition of the [http_sub_module][http_sub_module], nginx can perform appropriate substitution of the `cloud-config` `$private_ipv4` and `$public_ipv4` variables used to simplify network configuration. The `http_sub_module` is enabled in the official nginx binaries, and in most Linux distributions' nginx packages.